### PR TITLE
918621_docs_send_activity_notification_for_3p_(beta)

### PR DESCRIPTION
**Pull request by KirtiPereira**
Queries raised with Anand regarding the requirement. Clarifications awaited.

### CODE REVIEW

**Comment by laujan** [comment]
Please refrain from working on this PR until further notice.

File: api-reference/beta/api/team-sendactivitynotification.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+The following is an example of the request.
+
+<!-- {
+  "blockType": "request",
+  "name": "Send_activity_notification"
+}-->
+
+```http
+POST https://graph.microsoft.com/beta/teams/e5a548cb-e5a4-4e54-8da1-ec0e944d9c11/sendActivityNotificationPrivileged
```
Comment (satadrubiswasmsft) [inline api-reference/beta/api/team-sendactivitynotification.md:79]: The URL doesn't match the one above (`POST /teams/{id}/sendActivityNotification`)

File: api-reference/beta/api/user-sendactivitynotification.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+The following is an example of the request.
+
+<!-- {
+  "blockType": "request",
+  "name": "Send_activity_notification"
+}-->
+
+```http
+POST https://graph.microsoft.com/beta/users/629d12f1-c124-437e-b197-cc38f38cceb4/sendActivityNotification
```
Comment (satadrubiswasmsft) [inline api-reference/beta/api/user-sendactivitynotification.md:77]: URL doesn't match those listed above (`POST /users/{id}/teamwork/sendActivityNotification` and `POST /users/{id}/chats/{id}/sendActivityNotification`)

**Comment by laujan** [comment]
RESOLVED: https://github.com/microsoftgraph/microsoft-graph-docs/pull/10721

**Pull request closed without merging**